src/App/App.js

```diff
@@ -2,18 +2,25 @@ import React, { Component } from 'react';
 import ThumbnailContainer from '../Thumbnail-Container/ThumbnailContainer.js'
 import Header from '../Header/Header.js';
 import Movie from '../Movie/Movie.js';
-import { movieData } from '../movieData.js';
+import { getAllMovies, getMovieData } from '../apiCalls.js'
 import '../App/App.scss';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData,
-      selectedMovie: null
+      movies: [],
+      selectedMovie: null,
```

Comment:
Nice job setting initial values for movies and error! I would consider making selectedMovie an empty object.